fenix/app/src/main/java/org/mozilla/fenix/addons/InstalledAddonDetailsFragment.kt

```diff
@@ -425,12 +423,6 @@ class InstalledAddonDetailsFragment : Fragment() {
     }
 
     private fun SwitchMaterial.setState(checked: Boolean) {
```

Comment:
Nit: as `setState` doesn't have more behaviour lets remove it and use `isChecked` instead!